### PR TITLE
fix: slider widget issues

### LINKED ISSUE #18070: [Bug]: Category Slider UI consistency issues
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

1. In its default state category slider is wider than the other sliders
2. Marks in category slider are not visible by default like other sliders
Screen Shot 2022-11-01 at 10 08 54 PM

This should be worked on to make all sliders look and feel consistent.

### Steps To Reproduce

1. Drag and drop Category Slider, Number Slider and Range Slider
2. Observe the UI

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

1.8.6

### LINKED ISSUE #18194: [Bug]: Slider widget error message issues
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Two issues to tackle here:
1. Update the error message of the default start value in **_range and number slider_** -- Change to >= instead of > .
2. do the same for default end value -- Change to <= instead of <
Screen Shot 2022-11-16 at 4 06 52 PM

Screen Shot 2022-11-16 at 4 56 34 PM

### Steps To Reproduce

1. Dnd a slider widget(Range/number)
3. Set Min value 10 and default start value to 9 OR set max value =10 and default end value =11

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

production

### LINKED ISSUE #18272: [Feature]: Add tooltip property in slider widgets
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

- All slider widgets need a tooltip property in the general section.
- The logic should be same as the input widget logic
- "Tooltip always on" should be renamed to "Show value always"

### Why should this be worked on?

Consistency in the implementation of tooltips across Appsmith.

### PR DESCRIPTION
**Pull request by yaldram**
## Description

- Add tooltips to label and maintain consistency with other inputs.
- Change validation messages.
- Maintain consistency in widget sizes when dragged on the canvas.

Fixes #18272 
Fixes #18194 
Fixes #18070 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manual
- Cypress

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts [hunk 1]
```diff
@@ -296,7 +306,7 @@ export default [
       {
         propertyName: "tooltipAlwaysOn",
         helpText: "Keep showing the label with value",
-        label: "Tooltip Always On",
+        label: "Show value always",
```
Comment (keyurparalkar) [inline app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts:309]: @yaldram aren't we changing `tooltipAlwaysOn` value for category slider as well?

**Review by keyurparalkar — CHANGES_REQUESTED** [review]

File: app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts (hunk 1 above)
Comment (yaldram) [reply app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts:309]: We don't have `tooltip always on` property for category sliders.

Comment (keyurparalkar) [reply app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts:309]: @yaldram Should this be added? Can you confirm?
CC: @dilippitchika

Comment (yaldram) [reply app/client/src/widgets/RangeSliderWidget/widget/propertyConfig/contentConfig.ts:309]: Yes, I had discussed this with Dilip we removed it from the category slider.

**Review by keyurparalkar — APPROVED** [review]
LGTM!

**Comment by Sripriya93** [comment]
1. Tooltip property is present in General section
2. "Tooltip always on" is renamed to "Show value always"
3. Error message of the default start value in range and number slider is changed to "greater than or equal to" and default end value is "changed to less than or equal to"
4. In its default state category slider is now same as other sliders
5. Marks in category slider are now visible by default like other sliders

Moving to done

**Pull request merged**